datasets/recipes/minimal_recommended.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/",
+    "recordSet": "ml:RecordSet",
+    "field": "ml:Field"
+  },
+  "@type": "sc:Dataset",
+  "name": "Minimal example with recommended fields",
+  "description": "This is a minimal example, including the required and the recommended fields.",
+  "url": "https://example.com/dataset/minimal-recommended/about",
+  "citation": "@article{nash51, author = \"Nash, John\", title  = \"Non-cooperative Games\", journal = \"Annals of Mathematics\", year = 1951, volume = \"54\", number = \"2\", pages  = \"286--295\"}",
```

Comment:
I don't think we need to add citation into the "recommended" minimal example.

Reply:
+1

Reply:
I removed the citation. This does mean that we get a warning in the validator:
`[dataset(Minimal example with recommended fields)] Property "https://schema.org/citation" is recommended, but does not exist.`

Should the validator be updated so that this warning is not raised? @marcenacp